data-raw/create_strayr_dictionary.R

```diff
@@ -2,32 +2,51 @@
 library(tidyverse)
 
 state_table <- tibble::tribble(
-                                    ~state_name, ~state_abbr,     ~iso, ~postal,
-                              "New South Wales",       "NSW", "AU-NSW",   "NSW",
-                                   "Queensland",       "Qld", "AU-QLD",   "QLD",
-                              "South Australia",        "SA",  "AU-SA",    "SA",
-                                     "Tasmania",       "Tas", "AU-TAS",   "TAS",
-                                     "Victoria",       "Vic", "AU-VIC",   "VIC",
-                            "Western Australia",        "WA",  "AU-WA",    "WA",
-                 "Australian Capital Territory",       "ACT", "AU-ACT",   "ACT",
-                           "Northern Territory",        "NT",  "AU-NT",    "NT",
-                                    "Australia",       "Aus",     "AU",    "AU",
-                                        "Other",       "Oth", NA_character_, NA_character_,
-                                        "Total",       "Tot", NA_character_, NA_character_
-                 )
+  ~state_name, ~state_abbr,     ~iso, ~postal,
+  "New South Wales",       "NSW", "AU-NSW",   "NSW",
+  "Queensland",       "Qld", "AU-QLD",   "QLD",
+  "South Australia",        "SA",  "AU-SA",    "SA",
+  "Tasmania",       "Tas", "AU-TAS",   "TAS",
+  "Victoria",       "Vic", "AU-VIC",   "VIC",
+  "Western Australia",        "WA",  "AU-WA",    "WA",
+  "Australian Capital Territory",       "ACT", "AU-ACT",   "ACT",
+  "Northern Territory",        "NT",  "AU-NT",    "NT",
+  "Australia",       "Aus",     "AU",    "AU",
+  "Other Territories",       "Oth", NA_character_, NA_character_,
+  "Total",       "Tot", NA_character_, NA_character_
+)
+
+
+state_or_territory_palette <- c(
+  'New South Wales'    = '#87ceeb',
+  'Victoria'           = '#000080',
+  'Queensland'         = '#800000',
+  'South Australia'    = '#dc0014',
+  'Western Australia'  = '#fecb29',
+  'Tasmania'           = '#006a4e',
+  'Northern Territory' = '#e65a00',
+  'Australian Capital Territory' = '#94c949',
+  'Other Territories' = '#c0c0c0')
```

Comment:
should there be a colour for `"Australia"` -- maybe black? -- so it doesn't return `NA`?